polaris-core/src/main/java/org/apache/polaris/core/auth/ActivatedEntitySelector.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.core.auth;
+
+import java.util.Set;
+import java.util.function.Function;
+import org.apache.polaris.core.entity.PolarisBaseEntity;
+import org.apache.polaris.core.persistence.resolver.PolarisResolutionManifest;
+
+/**
+ * Defines which roles should be considered by authorization checks.
+ *
+ * @see PolarisAuthorizer
+ */
+public enum ActivatedEntitySelector {
```

Comment:
De we need this? It's just a wrapper around a function that takes a `PolarisResolutionManifest` as input. Since we only have 2 enum constants, couldn't you use a boolean instead?

Reply:
converted to `boolean`